king.yml

```diff
@@ -54,4 +55,4 @@
     - vars/ruby.yml
     - vars/matrix.yml
     - vars/monit.yml
-    - vars/letsencrypt.yml
+    - vars/letsencrypt_king.yml
```

Comment:
Maybe we could create `vars/{ ansible_hostname }/main.yml` for every host and let the `vars` be included automatically? Maybe Ansible can do this already.